public/javascripts/discover.js

```diff
@@ -92,6 +88,7 @@ function init(opts) {
         loadMoreAndApplyFilter,
         initial: numberOfSongsToShow,
         displayedSongs,
+        playlists,
         playlistName,
```

Comment:
Why does the searchBox still use playlistName?

Reply:
I think the searchBox don't need playlists either.
The searchBox works to search for songs.
Let me make sure of it.